src/v3/src/util/designTokens.ts

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright (c) 2023-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+export const DESIGN_TOKENS = {
+  // BorderColor
+  // BorderColorControl: '#8d8d8d',
```

Comment:
If these are intentionally commented out to show what tokens we _will_ support but only the uncommented ones are actually used right now it would be helpful to have a note here to that effect